panoptes_aggregation/extractors/shape_extractor.py

```diff
@@ -48,5 +48,9 @@ def shape_extractor(classification, **kwargs):
             if all(param in value for param in shape_params):
                 extract.setdefault(frame, {})
                 for param in shape_params:
-                    extract[frame].setdefault('{0}_{1}'.format(key, param), []).append(value[param])
+                    if value[param] is not None:
```

Comment:
Not blocking at all

Should we manually log these errors in sentry to know when they occur? https://docs.sentry.io/error-reporting/capturing/?platform=python#capturing-errors--exceptions

Doing so would allow us to detect these anomalous annotations submitted by our UI for further investigation by the team. Once the current backlog of old tess reductions clear this should only be for new subjects so only new annotation issues.